semantic-tsx/semantic-tsx.cabal

```diff
@@ -27,6 +27,7 @@ common haskell
                      , semantic-core ^>= 0.0
                      , semantic-source ^>= 0.0.1
                      , semantic-tags ^>= 0.0
+                     , semantic-diff
```

Comment:
I don’t know if we should be specifying version bounds here or not, but the difference between this and the preceding lines gives me pause. We need one for `semantic-source`, presumably, but maybe we shouldn’t specify them for _any_ of the local packages since we aren’t getting them from hackage? It only makes more work for us in keeping these up to date… if we ever update them. Which I suppose we won’t until we start moving them to hackage.

So I guess that makes this academic.